pkgs/tools/video/nordlicht/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, ffmpeg, popt, help2man }:
+    with cmake;
+    stdenv.mkDerivation rec {
+        name = "nordlicht-${version}";
+        version = "0.4.5-git.7ad3f008.4b1751bd";
+        license = "GNU GPL v2 or later";
+        description = "creates colorful barcodes from video files";
+        src = fetchFromGitHub {
+            owner  = "nordlicht";
+            repo   = "nordlicht";
+            rev    = "7ad3f008afe803037b332822028faed64b1751bd";
+            sha256 = "14657zfzvq0iki8sn6rq789bgjr7wvvid9y2clfy25bi0a1gvnix";
+        };
+        buildInputs = [ cmake ffmpeg popt help2man ];
```

cmake should be in nativeBuildInputs (isn't needed at runtime)
```suggestion
        nativeBuildInputs = [ cmake ];
        buildInputs = [ ffmpeg popt help2man ];
```

---

Pretty sure help2man belongs to nativeBuildInputs too (only needed to generate man pages at build time)